packages/vanilla/src/components/footer/footerNav.template.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * renders footer nav for tall
+ *
+ * @returns {object} JSX object
+ */
+function footerNavTemplate() {
+  return `
+  <nav data-autoid="dds--footer-nav" class="bx--footer-nav">
+    <ul class="bx--accordion bx--footer-nav__container">
```

Comment:
@andysherman2121 update all of the classname and stable selector prefixes here to pull from settings.